### PR TITLE
Fix: The date on the tooltip on the price chart is one day off

### PR DESCRIPTION
**Pull request by LukeSugiura**
### Outcome

Screen Shot 2020-02-27 at 12 04 46 PM

### CODE REVIEW

**Comment by valpinkman** [comment]
Graph is broken:

CleanShot 2020-02-27 at 16 17 15

File: src/renderer/components/Chart2/index.js [hunk 1]
```diff
@@ -115,28 +125,6 @@ const Chart = ({ height, data, color, renderTickY, renderTooltip, valueKey = "va
         display: false,
       },
       scales: {
-        xAxes: [
```
Comment (valpinkman) [inline src/renderer/components/Chart2/index.js]: why remove this ?

File: src/renderer/components/Chart2/index.js [hunk 2]
```diff
@@ -67,7 +67,15 @@ const ChartContainer: ThemedComponent<{}> = styled.div.attrs(({ height }) => ({
...
 
-const Chart = ({ height, data, color, renderTickY, renderTooltip, valueKey = "value" }: Props) => {
+const Chart = ({
+  height,
+  data,
+  color,
+  renderTickY,
+  renderTooltip,
+  valueKey = "value",
+  dateFormat = "MMM D",
```
Comment (juan-cortes) [inline src/renderer/components/Chart2/index.js]: Where is this used?

Comment (LukeSugiura) [reply src/renderer/components/Chart2/index.js]: Good question. I was wondering too. I saw it exists in Prop type but no component uses this prop. I'm just gonna delete it for now.

**Comment by juan-cortes** [comment]
I'd say squash them all together and we're good 😕

**Pull request merged**